InventoryBuilder.py

```diff
@@ -131,12 +132,8 @@ def token():
     def get_vrops(self):
         with open(self.json) as json_file:
             netbox_json = json.load(json_file)
-        vrops_list = list()
-        for target in netbox_json:
-            if target['labels']['job'] == "vrops":
-                vrops = target['labels']['server_name']
-                vrops_list.append(vrops)
-        self.vrops_list = vrops_list
+        self.vrops_list = [target['labels']['server_name'] for target in netbox_json if
```

Comment:
Like the idea, any specific reason for this change?

Reply:
I did it this way on the test and I changed it right here. No special reason, just some clean code.

Reply:
And it is faster.